src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_Basics/Task2.java

```diff
@@ -0,0 +1,23 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_Basics;
+
+import java.util.Scanner;
+
+public class Task2 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter a number please: ");
```

Comment:
To be more explicit, there is a range of numbers

Reply:
Done.